pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools>=42", "wheel"]
+requires = ["setuptools>=42", "wheel", "GitPython"]
```

Comment:
`GitPython` is not a build dependency, but an install dependencies. You need to add it here https://github.com/follnoob/extract-todo/blob/master/setup.cfg#L39

```
install_requires =
	GitPython
```

Reply:
Thanks! Fixed in ab753ed.